<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>output</class>
 <widget class="QMainWindow" name="output">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Team 7</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 225, 194);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exit_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>610</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(225, 242, 255);</string>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>610</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(225, 242, 255);</string>
    </property>
    <property name="text">
     <string>다시 하기</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output_text">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>1000</width>
      <height>270</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;나눔스퀘어라운드 Bold&quot;;
background-color: rgb(255, 234, 208);</string>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'나눔스퀘어라운드 Bold'; font-size:12pt; font-weight:72; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
